packages/template-retail-react-app/app/routes.jsx

```diff
@@ -71,6 +71,16 @@ export const routes = [
         component: ResetPassword,
         exact: true
     },
+    {
+        path: '/reset-password-landing',
+        component: ResetPassword,
+        exact: true
+    },
+    {
+        path: '/passwordless-login-landing',
+        component: Login,
+        exact: true
```

Comment:
Maybe using the constants file (at least for now) might be a good place to customize this. There will be a scenario where customers want to change this, and if they are using extensibility they probably don't want to override this routes or components to change the path strings.